airflow/migrations/versions/0050_3_0_0_add_dagbundlemodel.py

```diff
@@ -48,12 +49,16 @@ def upgrade():
         sa.Column("last_refreshed", UtcDateTime(timezone=True), nullable=True),
         sa.PrimaryKeyConstraint("name", name=op.f("dag_bundle_pkey")),
     )
+
     with op.batch_alter_table("dag", schema=None) as batch_op:
-        batch_op.add_column(sa.Column("bundle_name", sa.String(length=250), nullable=True))
+        batch_op.add_column(sa.Column("bundle_name", sa.String(length=250), nullable=False))
```

Comment:
We can't just set it nullable false here - we first have to populate it with data. Try this migration from 2.10 with some DAGs to see what I mean.

That means you will need an UPDATE statement like you had originally, you just didn't have it in the right spot.

Reply:
Oh, my bad, this is tricky, then. Should it be populated with `dags-folder` or `example-dags` or just any string? What if the old dag is from a git repo. I think best to update with dummy string and have the dag processor update it in the next cycle. 

@gyli , basically, you have to first set this column as nullable, then run the update query and alter the column to make it non-nullable